main.py

```diff
@@ -35,13 +32,17 @@ def init_logging(default_level=logging.INFO):
         logging.basicConfig(level=default_level)
     logging.info('Successfully initialized logger.')
 
+#move everything down so init_loggin is defined
```

Comment:
Please follow the comment format:
`# Start with capital letter, end with period.`